pages/api/sessions.ts

```diff
@@ -1,16 +1,11 @@
 // add api code here
 import { NextApiRequest, NextApiResponse } from 'next';
 import Stripe from 'stripe';
+import { stripeClient } from '../../utils/client';
 
-export type StripeSession = Stripe.Checkout.Session;
+export type StripeSession = Stripe.Checkout.Session & { url: string };
 export type EndpointResponse = { session: StripeSession } | { error: string };
 
-// 1. connect with stripe
-// auth with stripe
-const stripeClient = new Stripe(process.env.STRIPE_SECRET_KEY!, {
-  apiVersion: '2022-08-01',
-});
-
 export default async function handler(
   request: NextApiRequest,
```

Comment:
would be better to type the API requests (also for the REST API Design lecture on Friday):

https://github.com/upleveled/next-js-example-jan-2022/blob/5d8440eb00550a4c1156d81edfd8c4510cdc7e8e/pages/api/animals/index.ts#L8-L23